.coveragerc

```diff
@@ -3,4 +3,4 @@ command_line = -m unittest discover
 source =
     src/
 [report]
-fail_under = 82
+fail_under = 81
```

Comment:
We would rather increase the coverage not decrease, it would be best to add unit tests for this change

Reply:
Actually it was wrongly committed but this PR will create and delete ingress in a namespace.